update demo project (#93)
QGEP/docs: en/installation-guide/workstation.rst

* Restore the file `qgep_vx.x.x_structure_and_demo_data.backup` with pgAdmin

Install the demo project
------------------------

* Download from https://github.com/QGEP/qgep/releases/latest the qgep.zip

* Extract the file

> * If you do not remove the qgep_language.qm - file in the same directory as the qgep.qgs - file and start QGIS with language, your qgep-project will be translated if you start the project.

Reviewer: ```suggestion
* If you leave the qgep_[language].qm - file in the same directory as the qgep.qgs - file and start QGIS with language, your qgep-project will be translated if you start the project.
```
